lib/wadler/print.rb

```diff
@@ -75,17 +75,28 @@ def group(indent = 0, open_obj = '', close_obj = '',
     end
 
     # @param indent [Integer] nest indentation
+    # @param open_obj [String] nest oppening delimiter
+    # @param close_obj [String] nest closing delimiter
     # @param break_type [Oppen::Token::BreakType] nest breaking type
     #
     # @return [Nil]
-    def nest(indent, break_type = Oppen::Token::BreakType::CONSISTENT, &block)
+    def nest(indent, open_obj = '', close_obj = '',
+             break_type = Oppen::Token::BreakType::CONSISTENT, &)
```

Comment:
When using yield, one usually does not even mention the block at all.

Besides, make sure all this works in Ruby 3.1, which is, AFAICT, the current env in prod.

Reply:
The CI that Firas setup runs the tests on ruby 3.2 and 3.3. Should I add version 3.1?

Reply:
Yes.

Reply:
After verification, the README.md in langeod specifies that ruby 3.2+ is required. No need to add version 3.1 to the CI